bedrock/firefox/urls.py

```diff
@@ -77,8 +72,6 @@
         VariationTemplateView.as_view(template_name='firefox/features/private-browsing.html',
                                       template_context_variations=['a']),
         name='firefox.features.private-browsing'),
-    page('firefox/features/send-tabs', 'firefox/features/send-tabs.html'),
-    page('firefox/features/sync', 'firefox/features/sync.html'),
```

Comment:
It we're removing these page URLs, please make sure to update all references to pages that may be linking to them. For example, [this page](https://bedrock-integration-tests.oregon-b.moz.works/en-US/firefox/features/) is currently throwing a 500 because I'm guessing it contains a `{{ url('firefox.features.sync') }}` somewhere.

This is the cause of one of the test failures.